tests/e2e/device/e2e_device_commands.c

```diff
@@ -63,7 +63,6 @@ typedef struct E2E_TEST_COMMAND_STRUCT
 typedef E2E_TEST_COMMAND * E2E_TEST_COMMAND_HANDLE;
 /*-----------------------------------------------------------*/
 
-static const uint64_t ulGlobalEntryTime = 1639093301;

```

Comment:
@hihigupt do you know why this was originally done this way?

Reply:
Talked offline. This was done for simplicity and because RTC is platform specific. I will be making a change to the other samples and implementations in another PR.